docs/conf.py

```diff
@@ -0,0 +1,336 @@
+html_static_path = ["_static"]
+html_logo = "images/logo.png"
+html_sourcelink_suffix = ""
+html_title = "napari animation"
+
+favicons = [
+    {
+        # the SVG is the "best" and contains code to detect OS light/dark mode
+        "static-file": "favicon/logo-silhouette-dark-light.svg",
+        "type": "image/svg+xml",
+    },
+    {
+        # Safari in Oct. 2022 does not support SVG
+        # an ICO would work as well, but PNG should be just as good
+        # setting sizes="any" is needed for Chrome to prefer the SVG
+        "sizes": "any",
+        "static-file": "favicon/logo-silhouette-192.png",
+    },
+    {
+        # this is used on iPad/iPhone for "Save to Home Screen"
+        # apparently some other apps use it as well
+        "rel": "apple-touch-icon",
+        "sizes": "180x180",
+        "static-file": "favicon/logo-noborder-180.png",
+    },
+]
+
+html_css_files = [
+    "custom.css",
+]
+
+intersphinx_mapping = {
+    "python": ["https://docs.python.org/3", None],
+    "numpy": ["https://numpy.org/doc/stable/", None],
+    "napari_plugin_engine": [
+        "https://napari-plugin-engine.readthedocs.io/en/latest/",
+        "https://napari-plugin-engine.readthedocs.io/en/latest/objects.inv",
+    ],
+    "napari": [
+        "https://napari.org/dev",
+        "https://napari.org/dev/objects.inv",
+    ],
+}
+
+myst_enable_extensions = [
+    "colon_fence",
+    "dollarmath",
+    "substitution",
+    "tasklist",
+]
+
+myst_heading_anchors = 4
+suppress_warnings = ["etoc.toctree"]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = [
+    "_build",
+    "Thumbs.db",
+    ".DS_Store",
+    ".jupyter_cache",
+    "jupyter_execute",
+]
+
+# -- Examples gallery scrapers -------------------------------------------------
+
+
+class VideoScraper:
+    """Video file scraper class.
+    Scrapes video files that were saved to disk by the example scripts.
+    Based on the sphinx example scraper "Example 2: detecting image files on disk"
+    https://sphinx-gallery.github.io/stable/advanced.html#example-2-detecting-image-files-on-disk
+    """
+
+    def __init__(self):
+        self.seen = set()
+
+    def __repr__(self):
+        return "VideoScraper"
+
+    def __call__(self, block, block_vars, gallery_conf):
+        """Video file scraper.
+        Scrapes video files that were saved to disk by the example scripts.
+        Based on the sphinx example scraper "Example 2: detecting image files on disk"
+        https://sphinx-gallery.github.io/stable/advanced.html#example-2-detecting-image-files-on-disk
+        """
+        # Find all video files in the directory of this example.
+        video_file_extensions = [".mp4", ".mov"]
```

Comment:
I checked this works for .mp4 and .mov filetypes. We could add more, so long as someone checks they work well in the built docs.

Reply:
Although for some reason, when I build the documentation with `make docs` there are warnings output about the .mov file format. 

I'm not sure why, because the HTML website works as expected, regardless of whether we have .mov or .mp4 videos embedded. Regardless, I've changed the output files from our example scripts to use .mp4, just to avoid the warnings in the docs build.